site/app/models/gradeable/Gradeable.php

```diff
@@ -1163,4 +1177,11 @@ public function getAllGradingSections() {
 
         return $sections;
     }
-}
+    //return true if students can currently submit regrades for this assignment, false otherwise
+    public function isRegradeCurrentlyAllowed(){
```

Comment:
rename to `isRegradeOpen` to be consistent with `isSubmissionOpen`.  This should also check `isTaGradeReleased`